tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -866,6 +867,59 @@ public function testItThrowsWhenLookingUpIdentifierForUnknownEntity(): void
         $this->expectException(EntityNotFoundException::class);
         $this->_unitOfWork->getEntityIdentifier(new stdClass());
     }
+
+    public function testRemovedEntityIsRemovedFromManyToManyCollection(): void
+    {
+        $group       = new CmsGroup();
+        $group->name = 'test';
+        $this->_unitOfWork->persist($group);
+
+        $user       = new CmsUser();
+        $user->name = 'test';
+        $user->groups->add($group);
+        $this->_unitOfWork->persist($user);
+
+        $this->_unitOfWork->commit();
+
+        self::assertFalse($user->groups->isDirty());
+
+        $this->_unitOfWork->remove($group);
+        $this->_unitOfWork->commit();
+
+        // Test that the removed entity has been removed from the many to many collection
+        self::assertEmpty($user->groups);
```

Comment:
Your comments are great, why not use them as a second argument for the `assert*` calls?

```suggestion
        // Test that the removed entity has been removed from the many to many collection
        self::assertEmpty(
                $user->groups,
                'the removed entity should have been removed from the many to many collection'
        );
```